Etiquette/CODE_REVIEW.md

```diff
@@ -0,0 +1,109 @@
+* Focus not on finding flaws in the code (no one writes perfect code) but on understanding the change in the first place
+* Encourage things you like, try not only comment on what you don't like
+* Before suggesting to change code think if it brings any real value or if it is just your personal preference. Is this code something you can live with or not?
+* Do not try to make everything perfect (it never will be and otherwise we'll never deliver anything), but do not compromise on quality
+* Do not comment if you don't have any suggestions to address your comment
+* Use GitHub suggestions as much as possible
+* If you feel strong against the change - request changes and explain why you are doing that. Do not request changes for typos, missing documentation comments or code style violations, trust your colleagues to address them.
+* Approve changes if you are confident about them
+* Comment on the changes if something is not clear to you and needs further clarification or actions
+* If you are not familiar with the domain ask for more context (better in person) rather than skipping to something more familiar
+* Ask, don't tell. ("What do you think about...?", "How about ...?", "Should we try ...?")
+* Be brief but clear in your comments (code review is not about proving who is more correct or knows more)
+* Unless explicitly asked for it do not suggest code to be written in a specific way - let the author to find or suggest a solution themselves, they may come up with a better one
+* Submit all the comments as a single review, even if there is just one comment
+* If you see a stale pull request without any updates - ask for the status update in the comment
+* Do not be a blocker - if you requested changes and they were addressed make sure you review it
+* Do not merge others pull requests - they may be waiting for some one else review
+
+Responding to feedback
+-------------------------
+
+* Wait before answering to the single comments, reviewer may have more (though they should prefer to send all comments at once)
+* Be grateful for the reviewer's suggestions. ("Good call. I'll make that
+  change.")
+* If you disagree with suggestion explain way and avoid template answers like "out of the scope", "thank you for your opinion". If you think that suggestion is a personal preference and does not bring value - say so
+* Assume the best intention from the reviewer's comments.
+* If a reviewer seems aggressive or angry or otherwise personal, consider if it is intended to be read that way and ask the person for clarification of intent, in person if possible. Do not escalate online discussions.
+* If you can't reach agreement seek for more opinions
+* Don't be too defensive about your code, someone will rewrite it anyway sooner or later (it might be yourself as well)
+* Try to answer all comments. You can review your own code for that instead of sending individual replies
```

Comment:
```suggestion
* Try to answer all comments, don't make teammates feel like you are ignoring them. You can review your own code for that instead of sending individual replies so that all your answers are posted at once.
```